README.md

```diff
@@ -2,27 +2,27 @@
 
 !["amelia robot logo"](graphics/logo.png)
 
-Click [here](https://github.com/apps/registratortest/installations/new) to install.
-
 Registrator is a GitHub app that automates creation of registration pull requests for your julia packages to the [General](https://github.com/JuliaRegistries/General) registry.
 
+Click [here](https://github.com/apps/registratortest/installations/new) to install the app.
```

Comment:
a button from shields.io would be nice, I made one [here](https://github.com/JuliaComputing/Registrator.jl/issues/46#issuecomment-479455721)